### ISSUE TITLE
`fromImage` doesn't inherit parent image environment variables

### ISSUE THREAD

**Issue by jlesquembre** [nlewo/nix2container#79]
With nixpkgs `dockerTools`, environment variables defined in a parent image are inherited with `fromImage`: 
https://github.com/NixOS/nixpkgs/blob/b1da47faa15bb0ce041b9acdd64e7138bffa7646/pkgs/build-support/docker/examples.nix#L309-L311

That's not the case in nix2container. Other configuration in the parent image, like `entrypoint` or `workingDir`, aren't inherited neither (but that's also the case with nixpkgs `dockerTools`)

**Comment by nlewo** [comment]
Correct, and this is a known limitation: https://github.com/nlewo/nix2container/blob/95e2220911874064b5d809f8d35f7835184c4ddf/nix/image.go#L150

I don't know when i could implement this but PR are welcome ;)
